app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -63,18 +63,18 @@ sealed class NavigationItem(
 
     object Conversation : NavigationItem(
         route = "conversation/{$CONVERSATION_ID_ARGUMENT}",
-        content = {
-            ConversationScreen(hiltViewModel())
-        },
+        content = { ConversationScreen(hiltViewModel()) },
         arguments = listOf(
             navArgument(CONVERSATION_ID_ARGUMENT) { type = NavType.StringType }
         )
     ) {
-        fun createRoute(conversationId: ConversationId) = "conversation/${conversationId}"
+        fun createRoute(conversationId: ConversationId) = "conversation/${conversationId.domain}@${conversationId.value}"
```

Comment:
We may need this somewhere else. How about an extension  function that do this concatenation ?